### PR TITLE
feat: include and exclude in copy info

### PR DESCRIPTION
**Pull request by morlay**
related: https://github.com/dagger/dagger/issues/2711

Then we could get readable image history. 

image

### CODE REVIEW

File: plan/task/copy.go
Comment (TomChv) [inline plan/task/copy.go:91]: Could you define this type above, it's easier to understand then

**Review by TomChv — APPROVED** [review]
LGTM, I left a minor style comment but that looks good.
Let's wait for @sipsma or @helderco approval before merge this one.

**Comment by jpadams** [comment]
cc @helderco

**Comment by gerhard** [comment]
While the code makes sense, I cannot visualise the improvement.

Can you share a simple example @morlay we can run to see the behaviour before & after this change? I can see this helping future maintainers too.

Alternatively, it's OK to wait for @helderco & @sipsma which may have the context and can share.

**Comment by morlay** [comment]
it could be found in logs when #Copy used.

For final build history display, have to wait for this
https://github.com/moby/buildkit/pull/2932

**Pull request closed without merging**
